app/gemini-bu-rot/app.toml

```diff
@@ -99,7 +99,7 @@ pins = [
 name = "drv-lpc55-swd"
 priority = 4
 max-sizes = {flash = 16384, ram = 4096}
-uses = ["flexcomm3"]
+uses = ["flexcomm3", "iocon"]
```

Comment:
This means that both `swd` and `gpio_driver` can use `iocon` – any concerns about that sharing?  It seems okay, since the only thing they do is write to an address derived from `device::IOCON::ptr()`